docs/plugins/memcached.md

```diff
@@ -0,0 +1,35 @@
+# `memcached` plugin
+
+The `memcached` plugin consumes [Memcached](https://memcached.org/) journald log entries from the local filesystem and outputs parsed entries.
```

Comment:
Technically the logs exist on the filesystem but we read them from journalctl's output, not from the files themselves.

Reply:
Expanding on this. Memcached logs to sdout / stderr, which is captured by systemd and written to the "journal". We use journalctl to output json logs (captured by systemd).

Reply:
updated description for better clarity